Lib/test/test_tools/test_pathfix.py

```
@@ -39,30 +39,62 @@ def test_pathfix(self):
+                ['-i', '/usr/bin/python3']),
             '#! /usr/bin/python3',
         )
         self.assertEqual(
             self.pathfix(
                 '#! /usr/bin/env python -R',
-                ['-i', '/usr/bin/python3', ]),
+                ['-i', '/usr/bin/python3']),
             '#! /usr/bin/python3',
         )
 
     def test_pathfix_keeping_flags(self):
         self.assertEqual(
             self.pathfix(
                 '#! /usr/bin/env python -R',
-                ['-i', '/usr/bin/python3', '-k',]),
+                ['-i', '/usr/bin/python3', '-k']),
             '#! /usr/bin/python3 -R',
         )
         self.assertEqual(
             self.pathfix(
                 '#! /usr/bin/env python',
-                ['-i', '/usr/bin/python3', '-k',]),
+                ['-i', '/usr/bin/python3', '-k']),
             '#! /usr/bin/python3',
         )
 
+    def test_pathfix_adding_flag(self):
+        self.assertEqual(
+            self.pathfix(
+                '#! /usr/bin/env python',
+                ['-i', '/usr/bin/python3', '-a', 's']),
+            '#! /usr/bin/python3 -s',
+        )
+        self.assertEqual(
+            self.pathfix(
+                '#! /usr/bin/env python -s',
+                ['-i', '/usr/bin/python3', '-a', 's']),
+            '#! /usr/bin/python3 -s',
+        )
+        self.assertEqual(
+            self.pathfix(
+                '#! /usr/bin/env python -v',
+                ['-i', '/usr/bin/python3', '-a', 'v', '-k']),
+            '#! /usr/bin/python3 -vv',
+        )
+        self.assertEqual(
+            self.pathfix(
+                '#! /usr/bin/env python',
+                ['-i', '/usr/bin/python3', '-a', 'Rs']),
+            '#! /usr/bin/python3 -Rs'
+        )
+        self.assertEqual(
+            self.pathfix(
+                '#! /usr/bin/env python -W something',
+                ['-i', '/usr/bin/python3', '-a', 's', '-k']),
+            '#! /usr/bin/python3 -sW something',
+        )
```

What if -a argument contains a space? Should we raise an error?